package.json

```diff
@@ -33,6 +33,7 @@
   },
   "homepage": "https://github.com/react-navigation/react-navigation-stack#readme",
   "dependencies": {
+    "react-native-screens": "^1.0.0-alpha.3"
```

Comment:
we should probably move this into react-navigation itself since it will be depended on by other navigators, then make it a peer dependency here